packages/react-native-web/src/exports/Modal/__tests__/ModalAnimation.js

```diff
@@ -0,0 +1,173 @@
+/* eslint-env jasmine, jest */
+
+import ModalAnimation from '../ModalAnimation';
+import React from 'react';
+import { cleanup, render } from '@testing-library/react';
+
+describe('components/Modal/ModalAnimation', () => {
```

Comment:
Please combine all the tests into a single test file that is testing the public API of `Modal` rather than individual components that are part of the current implementation

Reply:
Ok - I had to drop a few of the tests to be able to test against only the public API,  Some of the tests are also no longer testing the functionality as intended (eg, the focus trap tests)